.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: build-and-test
+on: [push]
+jobs:
+  build-and-test:
```

Comment:
What about we mention "namespace" or at least "darwin" in the workflow and job names here?